cityofaustin/janis
# 4404 News list page

# Description
Added:
* News list page
* News list on department page
* Links to news list page from news page




https://github.com/cityofaustin/techstack/issues/4404



https://github.com/cityofaustin/joplin/pull/736

# Types of changes



# How can this be tested?



 
     
https://janis-v3-4404-news-pag.netlify.app/

Cupcake news is written by aph for aph
Cupcake court is written by muni court for aph
Bacon court is written by muni court for muni court

Confirm no more than 3 news items appear on the department page: https://janis-v3-4404-news-pag.netlify.app/en/austin-public-health/

Check the news list page (that appears per dept): https://janis-v3-4404-news-pag.netlify.app/en/austin-public-health/news/#page=1, https://janis-v3-4404-news-pag.netlify.app/en/municipal-court/news/#page=1

News pages have link to news list page in right rail:
https://janis-v3-4404-news-pag.netlify.app/en/austin-public-health/court-cupcakes/


# Checklist:
